cmdapp/src/main.rs

```diff
@@ -68,45 +68,45 @@ pub fn config_from_args() -> (PathBuf, PathBuf, Config) {
             .long("filter_speckle")
             .short("f")
             .takes_value(true)
-            .help("Discard patches smaller than X px in size"),
+            .help("Discard patches smaller than X px in size. Accepts integer values within the range [0, 16](Default: 4)"),
     );
 
     let app = app.arg(
         Arg::with_name("color_precision")
             .long("color_precision")
             .short("p")
             .takes_value(true)
-            .help("Number of significant bits to use in an RGB channel"),
+            .help("Number of significant bits to use in an RGB channel. Accepts integer values within the range [1, 8](Default: 6)"),
     );
 
     let app = app.arg(
         Arg::with_name("gradient_step")
             .long("gradient_step")
             .short("g")
             .takes_value(true)
-            .help("Color difference between gradient layers"),
+            .help("Color difference between gradient layers. Accepts integer values within the range [0, 255]"),
```

Comment:
```suggestion
            .help("Color difference between gradient layers. Accepts integer values within the range [0, 255](Default: 16)"),
```

Reply:
Yeah strange that `gradient_step` maps to `layer_difference`